src/main/java/com/shevtsov/dao/ClientDao.java

```diff
@@ -3,5 +3,41 @@
 import com.shevtsov.domain.Client;
 
 public interface ClientDao {
+
+    /**
+     * Saves Client-object to DB.
+     * @param client the Client-object, which is saved.
+     * @return true if new client was saved successfully, or false otherwise.
+     */
     boolean saveClient(Client client);
+
+    /**
+     * Modifies client data in DB.
+     * @param id id of client, which is modified.
+     * @param newName id of client, which is modified.
+     * @param newSurname new name of client, which is modified.
+     * @param newPhone new surname of client, which is modified.
+     * @return true if client was modified successfully, or false otherwise.
+     */
+    boolean modifyClient(long id, String newName, String newSurname, String newPhone);
+
+    /**
+     * Removes client in DB.
+     * @param client the Client-object, which is removed.
+     * @return true if client was removed successfully, or false otherwise.
+     */
+    boolean removeClient(long client);
+
+    /**
+     * Creates collection of Client-objects from DB and transmits it to Source for processing.
+     */
+    void createClientsList();
```

Comment:
this method could be very dirty, because it create and send something, but nothing to return

Reply:
changed to List<Client> gatAll();